src/main/java/com/librarymanagement/application/backend/controller/MemberController.java

```diff
@@ -0,0 +1,52 @@
+package com.librarymanagement.application.backend.controller;
+import com.librarymanagement.application.backend.dto.Member;
+import com.librarymanagement.application.backend.service.MemberService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+public class MemberController {
+    final MemberService memberService;
+
+    public MemberController(@Autowired MemberService memberService) {
```

Comment:
why can't we just autowire a service?

Reply:
We could do that. I thought I read somewhere as it is not recommended.